FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -137,8 +161,18 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
 
                     _result.SequenceIdToJobIdMapForRunningJobs[input.ProcessingJobSequenceId] = newJobId;
 
-                    // if enqueue successfully while fails to report result, will re-enqueue and return the existing jobInfo
-                    progress.Report(JsonConvert.SerializeObject(_result));
+                    if (input.SplitParameters != null)
+                    {
+                        foreach (var param in input.SplitParameters)
+                        {
+                            _result.SubmittedResourceTimestamps[param.Key] = param.Value.DataEndTime;
+                        }
+                    }
+
+                    _result.SubmittingProcessingJob = null;
```

Comment:
do we need assign the SubmittingProcessingJob field for group scope?

Reply:
Yes, both group and system scope need assign this variable to null after enqueue.